build/common.mk

```diff
@@ -104,7 +104,11 @@ endif
 PULUMI_BIN          := $(PULUMI_ROOT)/bin
 PULUMI_NODE_MODULES := $(PULUMI_ROOT)/node_modules
 
-.PHONY: default all ensure only_build only_test build lint install test_fast test_all core
+GO_TEST_FAST = go test -short -v -count=1 -cover -timeout 2h -parallel ${TESTPARALLELISM}
+GO_TEST = go test -v -count=1 -cover -timeout 2h -parallel ${TESTPARALLELISM}
+GOPROXY = 'https://proxy.golang.org'
+
+.PHONY: default all ensure only_build only_test build lint install test_all core
```

Comment:
I assume this is all the standard common.mk and this was justbout of date previously?

Reply:
yessir - exactly that :(